File: cp/cd.py
import sys
for i in range(int(input())):
    n = int(input())
    power = pow(10 , n) 
    a = int(input())
    s = 2*power 
    print(s + a)
    sys.stdout.flush()
    b = int(input())
    s -= power 
    print(power - b)
    sys.stdout.flush()
    d = int(input())
    s -= power
    print(power - d)
    sys.stdout.flush()
    if '-1' == input():
        exit(0)

File: cp/cd.cpp
#include<bits/stdc++.h>
using namespace std ;
int main(){
    int n_t ;
    cin>>n_t ;
    int a ,  b , d ,n ;
    int result ;
    double s ;
    for(int i =0  ; i < n_t ; i++){
        cin>>n ;
        cin>>a ;
        s = 5*pow(10 , n);
        cout<<s ;
        cout<<s<<endl;
        cout.flush();
        s -= (float) a ;
        cin>>b ;
        s -= (float) b + 1 ;
        cout<< 1 ;
        cout.flush();
        cin>>d ;
        s -= (float) d + 1 ;
        cout<< s ;
        cout.flush();
        cin>>result ;
        if (result == -1)
            return 0; 
    }
}
